Src/Plugins/Security/SEC101_003.GoogleApiKeyValidator.cs

```diff
@@ -58,12 +58,15 @@ public static string IsValidDynamic(ref string fingerprint, ref string message)
         protected override string IsValidDynamicHelper(ref string fingerprintText,
                                                        ref string message)
         {
+            const string Invalid = "RequestDenied: The provided API key is invalid";
+            const string Expired = "RequestDenied: The provided API key is expired";
+            const string RevokedKey = "RequestDenied: Google has disabled the use of APIs from this API project";
+            const string ProjectNotAuthorized = "RequestDenied: This API project is not authorized to use this API";
+            const string KeyNotAuthorized = "RequestDenied: This API key is not authorized to use this service or API";
+            const string RefererRestrictions = "RequestDenied: API keys with referer restrictions cannot be used with this API";
+            const string IpNotAuthorized = "RequestDenied: This IP, site or mobile application is not authorized to use this API key";
```

Comment:
>IpNotAuthorized [](start = 25, length = 15)

The right casing of this is 'IPNotAuthorized'. It looks weird, but .NET guidelines state that a two-letter acronyms is all caps. :)